Frends.Facebook.Request/Frends.Facebook.Request.Tests/UnitTests.cs

```diff
@@ -62,53 +62,56 @@ public void TestGetInsights()
     {
         var input = new Input
         {
+            Method = Methods.GET,
             QueryParameters = "metric=page_impressions_unique&metric=post_reactions_love_total",
             Reference = "insights",
             AccessToken = token,
             ApiVersion = "18.0",
         };
 
-        var ret = Facebook.Get(input, new Options(), default);
+        var ret = Facebook.Request(input, new Options(), default);
         Assert.IsNotNull(ret);
 
-        // Assert.IsTrue(ret.Result.Success);
+        // Assert.IsTrue(ret.Result.Statuscode == 200);
```

Comment:
Please remove unnecessary comments.

Reply:
I updated the comments to be a bit more informative. Comments are needed in case the user is added the required permissions and the tests need to be updated.